src/main/resources/public/raml/resourceTypes/aggregated_object.raml

```diff
@@ -0,0 +1,28 @@
+description: This class returns aggregated data given a specific ID of an aggregated object.
+
+/{id}:
+  uriParameters:
+    id:
+      displayName: id
+      description: Id of the aggregatedObject
+      required: true
```

Comment:
id should not be required since you may want to fetch all aggregated objects.

Reply:
id is required

Reply:
I don't think we currently support getting the whole collection.